src/core/audit-fd.c

```diff
@@ -37,6 +38,13 @@ static int audit_fd;
 int get_audit_fd(void) {
 
         if (!initialized) {
+                if (have_effective_cap(CAP_AUDIT_WRITE) == 0) {
+                        audit_fd = -1;
```

Comment:
Agreed and noted; we're only concerned with audit_fd being a negative error value anyway.  Will update and rebase shortly.